pypeit/core/wavecal/autoid.py

```diff
@@ -315,13 +317,9 @@ class General:
       2D array of arcline spectra (nspec,nslit)
     lines : list
       List of arc lamps on
+    par:
```

Comment:
Add `ParSet`

Reply:
And I guess it is `['calibrations']['wavelengths']`
or something like that.

Reply:
I put in Parset or dict.